README.md

```diff
@@ -34,6 +34,11 @@ let app = new EmberApp(defaults, {
     }
 ```
 
+From `ember-cli-uniq@0.3` to `ember-cli-uniq@0.4`:
+
+* Replace bower import of `uniq` to npm import of `uniplaces-uniq`
+* Replace usage of `inline-svg` with `svg-jar`
```

Comment:
How? By replacing every occurence/invocation of the {{inline-svg}} helper with the {{svg-jar}} helper :)